### PR TITLE
fix: broken link

### PR DESCRIPTION
**Pull request by joachimprinzbach**

### CODE REVIEW

File: content/master/software/install.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ Below are vendors providing conformant Crossplane distributions.
 #### Upbound
 Upbound, the founders of Crossplane, maintains a free and open source 
 distribution of Crossplane called 
-[Universal Crossplane](https://www.upbound.io/products/universal-crossplane)
+[Universal Crossplane](https://www.upbound.io/product/universal-crossplane)
```
Comment (jbw976) [inline content/master/software/install.md:335]: this change looks correct to me now and is working on the preview site. nice find!

can you also make this same change to https://github.com/crossplane/docs/blob/master/content/v1.16/software/install.md, so folks will see your update right away on the latest (i.e. v1.16) version of the site? 😁

**Review by jbw976 — COMMENTED** [review]
awesome, thanks for taking the time to make this update @joachimprinzbach! It's functionally correct, but just one request for you to make this same change to the v1.16 directory too 🙇‍♂️

**Comment by jbw976** [comment]
I think your commit may also need a DCO sign-off, which can be easily updated with these instructions: https://github.com/crossplane/crossplane/tree/master/contributing#certificate-of-origin

File: content/master/software/install.md (hunk 1 above)
Comment (joachimprinzbach) [reply content/master/software/install.md:335]: Included the fix in all the versions

**Comment by joachimprinzbach** [comment]
> I think your commit may also need a DCO sign-off, which can be easily updated with these instructions: https://github.com/crossplane/crossplane/tree/master/contributing#certificate-of-origin

Yeah thanks for pointing out - commit is signed now

**Review by jbw976 — APPROVED** [review]
LGTM @joachimprinzbach, thank you for backporting this fix to previous versions as well! 💪

**Pull request merged**